src/api-reference/cards/v4.cards-get-started.markdown

```diff
@@ -14,14 +14,16 @@ The Cards v4 API exposes the following resources:
 Resource|Description
 ---|---
 Account|Partners can create Accounts via bulk API.
-Transaction|Partners can create Transactions via bulk API.
+Transaction|Partners can create Transactions via bulk API. 
```

Comment:
Added space at the end on purpose?

Reply:
Was not on purpose - removed: [4a8e94d](https://github.com/SAP-docs/preview.developer.concur.com/pull/1169/commits/4a8e94d8cbe654d9ad3f1f02573f200199130330)